### PR TITLE
Add Google Colab

### LINKED ISSUE #3313: Add Google Colab icon!
**Name:** Google Colab
**Website:** [Google Colab](https://colab.research.google.com/)
**Alexa rank:**  Rank is number 1, as its a google service.

**Official resources for icon and color:** [ICON](https://colab.research.google.com/img/colab_favicon_256px.png)

### PR DESCRIPTION
**Pull request by fbernhart**
googlecolab

Closes #3313 
**Alexa rank:** Not working for https://colab.research.google.com, as it's a subdomain of google.com. BUT as discussed in #3313 Google Colab is popular enough to be added.

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- **Source:** Embedded SVG from the header at https://colab.research.google.com
- **Color:** I've used `#F9AB00`, which is one of the two main colors which can be found in the logo. The other one is `#E8710A`. I've used the first one, as it's used for "three parts in the logo". Two times in the "C" and once in the "O". Unfortunately I haven't found any style guides or anything else. As well their [website](https://colab.research.google.com) doesn't have really any other elements in those colors. So `#E8710A` might be just as good. Let me know if you have any better idea. 😊 

- **Gaps:** I've used a width of 0.35px to create the gaps to separate the color changes.
- **Precision:** A precision of 4 was unfortunatelly necessary. I wasn't able to pass the linter locally with a precision of 3.

### CODE REVIEW

**Comment by rain1024** [comment]
Hope this PR will be merged soon.

File: icons/googlecolab.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Google Colab icon</title><path d="M16.9414 4.9757a7.033 7.033 0 0 0-4.9308 2.0646 7.033 7.033 0 0 0-.1232 9.8068l2.395-2.395a3.6455 3.6455 0 0 1 5.1497-5.1478l2.397-2.3989a7.033 7.033 0 0 0-4.8877-1.9297zM7.07 4.9855a7.033 7.033 0 0 0-4.8878 1.9316l2.3911 2.3911a3.6434 3.6434 0 0 1 5.0227.1271l1.7341-2.9737-.0997-.0802A7.033 7.033 0 0 0 7.07 4.9855zm15.0093 2.1721l-2.3892 2.3911a3.6455 3.6455 0 0 1-5.1497 5.1497l-2.4067 2.4068a7.0362 7.0362 0 0 0 9.9456-9.9476zM1.932 7.1674a7.033 7.033 0 0 0-.002 9.6816l2.397-2.397a3.6434 3.6434 0 0 1-.004-4.8916zm7.664 7.4235c-1.38 1.3816-3.5863 1.411-5.0168.1134l-2.397 2.395c2.4693 2.3328 6.263 2.5753 9.0072.5455l.1368-.1115z"/></svg>
```
Comment (adamrusted) [inline icons/googlecolab.svg:1]: Icon looks great - any chance of reducing it down to 3 decimals though please? 😅

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/googlecolab.svg (hunk 1 above)
Comment (fbernhart) [reply icons/googlecolab.svg:1]: @adamrusted No, unfortunately not.

> Precision: A precision of 4 was unfortunatelly necessary. I wasn't able to pass the linter locally with a precision of 3.

**Review by adamrusted — APPROVED** [review]
LGTM!

**Pull request merged**